classes/Hook.php

```diff
@@ -102,6 +102,14 @@ class HookCore extends ObjectModel
         'displayProductTabContent' => ['from' => '1.7.0.0'],
         'displayProductTab' => ['from' => '1.7.0.0'],
 
+        // Order page
+        'displayAdminOrderRight' => ['from' => '1.7.7.0'],
+        'displayAdminOrderLeft' => ['from' => '1.7.7.0'],
+        'displayAdminOrderTabOrder' => ['from' => '1.7.7.0'],
+        'displayAdminOrderTabShip' => ['from' => '1.7.7.0'],
+        'displayAdminOrderContentOrder' => ['from' => '1.7.7.0'],
+        'displayAdminOrderContentShip' => ['from' => '1.7.7.0'],
+
```

Comment:
@matks We cannot use ` AppKernel::VERSION` 🤔
See https://github.com/PrestaShop/PrestaShop/blob/1.7.7.x/app/AppKernel.php#L33

Reply:
No because in the next versions `AppKernel::VERSION` will change (1.7.8, 1.7.8, ...) So this information needs to be hard coded so that it never changes

Reply:
@jolelievre  but the next version (1.7.8.x for example) is from the develop branch and not 1.7.7.x ?

Reply:
Yes so when we will merge this branch into develop, the const value will have changed But this class will still need to know which exact version deprecated these hooks

Reply:
You are right ! Thank you